MembraneRTC/src/main/java/org/membraneframework/rtc/InternalMembraneRTC.kt

```diff
@@ -150,7 +175,14 @@ constructor(
             screencastTrack.start()
         }
 
-        pc.addTrack(screencastTrack.rtcTrack(), listOf(UUID.randomUUID().toString()))
+        if(simulcastConfig.enabled) {
+            val sendEncodings = getSendEncodingsFromConfig(simulcastConfig)
+            val transceiverInit =
+                RtpTransceiverInit(RtpTransceiverDirection.SEND_ONLY, listOf(UUID.randomUUID().toString()), sendEncodings)
+            pc.addTransceiver(screencastTrack.rtcTrack(), transceiverInit)
+        } else {
+            pc.addTrack(screencastTrack.rtcTrack(), listOf(UUID.randomUUID().toString()))
```

Comment:
Maybe we could use `addTransceiver`. This way we will get the same code as in 238-246